M2/Macaulay2/packages/CoincidentRootLoci/examples/___Coincident__Root__Locus_sp_st_sp__Coincident__Root__Locus.out

```diff
@@ -16,14 +13,14 @@ o2 = CRL(13,1,1)
 o2 : CoincidentRootLocus
 
 i3 : time X * Y
-     -- used 0.531738 seconds
+     -- used 0.529559 seconds
```

Comment:
Please don't use the "time" command in any examples in packages whose example output is cached, for then if we install the external program on our machine and rerun the examples, the cached output will change in a trivial way, preventing us from determining easily whether it changed in a nontrivial way.  Also, it means your periodic updates contain those trivial changes.

Reply:
You are right. Now I remove them.